src/cmd/integration.go

```diff
@@ -57,16 +68,42 @@ spec:
   name: "Prod"
   iamRole: "arn:aws:iam::XXXXX:role/opslevel-integration"
   externalId: "XXXXXX"
-  ownershipTagOverrides: true
+  ownershipTagOverride: true
```

Comment:
```suggestion
  awsTagsOverrideOwnership: true
```

Needs to match the tag in opslevel-go, see here https://github.com/OpsLevel/opslevel-go/blob/408867194d2bb24daf1627d8cb759fdfc24f9089/integration.go#L45